### PR TITLE
Close #2928 Always show summary fields when adding or editing content.

### LINKED ISSUE #2928: Always show summary fields when adding or editing content.
## User Story(s)
As a content editor or content admin, when adding or editing news, and other content types that come with Quickstart, it is not apparent that the Summary fields are very important fields.

It would make sense to me to have the summary section open by default when adding or editing content.

Slightly customized default
image

Proposed
image

### PR DESCRIPTION
**Pull request by trackleft**
## Description
When editing content it is not apparent just how important the summary fields are.

I'v set the summary fieldset to always open on the edit form of each content type that comes with quickstart.
This has been done for 
- az_news
- az_event
- az_flexible_page
- az_person

## Related issues
Closes #2928 

## How to test
Create or edit any of the following types of content: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2932.probo.build/user/login?destination=/node/add

- az_news
- az_event
- az_flexible_page
- az_person

Notice the summary section on the right of the screen.
image
image
image
image

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [x] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by sanjanans — DISMISSED** [review]

**Review by sanjanans — APPROVED** [review]

File: modules/custom/az_event/config/install/core.entity_form_display.node.az_event.default.yml [hunk 1]
```diff
@@ -42,11 +42,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: ''
+        show_empty_fields: false
         id: ''
         open: true
-        description: 'For use in feeds and content listings.'
+        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (bberndt-uaz) [inline modules/custom/az_event/config/install/core.entity_form_display.node.az_event.default.yml:48]: I think the first sentence is sufficient for all the content types. This description could also link to a quickstart.arizona.edu page on content summaries once that is ready.

```suggestion
        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml [hunk 2]
```diff
@@ -49,11 +49,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: 'group-summary field-group-tab'
+        show_empty_fields: false
         id: ''
-        open: false
+        open: true
+        description: 'Summary fields determine how this news appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (bberndt-uaz) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml:55]:
```suggestion
        description: 'Summary fields determine how this news article appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```

File: modules/custom/az_flexible_page/config/install/core.entity_form_display.node.az_flexible_page.default.yml [hunk 3]
```diff
@@ -28,11 +28,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: ''
+        show_empty_fields: false
         id: ''
-        open: false
-        description: 'These fields appear in display modes other that the full page view mode.'
+        open: true
+        description: 'Summary fields determine how this page appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_flexible_page/config/install/core.entity_form_display.node.az_flexible_page.default.yml:34]:
```suggestion
        description: 'Summary fields determine how this page appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_news/config/install/core.entity_form_display.node.az_news.default.yml [hunk 4]
```diff
@@ -47,11 +47,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: 'group-summary field-group-tab'
+        show_empty_fields: false
         id: ''
-        open: false
+        open: true
+        description: 'Summary fields determine how this news article appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_news/config/install/core.entity_form_display.node.az_news.default.yml:53]:
```suggestion
        description: 'Summary fields determine how this news article appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_person/config/install/core.entity_form_display.node.az_person.default.yml [hunk 5]
```diff
@@ -56,10 +56,10 @@ third_party_settings:
         classes: 'group-summary field-group-tab'
         show_empty_fields: false
         id: ''
-        open: false
-        description: ''
+        open: true
+        description: 'Summary fields determine how this person appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_person/config/install/core.entity_form_display.node.az_person.default.yml:60]:
```suggestion
        description: 'Summary fields determine how this person appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_event.default.yml [hunk 6]
```diff
@@ -44,11 +44,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: ''
+        show_empty_fields: false
         id: ''
         open: true
-        description: 'For use in feeds and content listings.'
+        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_event.default.yml:50]:
```suggestion
        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_flexible_page.default.yml [hunk 7]
```diff
@@ -30,11 +30,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: ''
+        show_empty_fields: false
         id: ''
-        open: false
-        description: 'These fields appear in display modes other that the full page view mode.'
+        open: true
+        description: 'Summary fields determine how this page appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_flexible_page.default.yml:36]:
```suggestion
        description: 'Summary fields determine how this page appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml [hunk 8]
```diff
@@ -49,11 +49,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: 'group-summary field-group-tab'
+        show_empty_fields: false
         id: ''
-        open: false
+        open: true
+        description: 'Summary fields determine how this news article appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml:55]:
```suggestion
        description: 'Summary fields determine how this news article appears in feeds, listings and other displays throughout the site.'
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_person.default.yml [hunk 9]
```diff
@@ -58,10 +58,10 @@ third_party_settings:
         classes: 'group-summary field-group-tab'
         show_empty_fields: false
         id: ''
-        open: false
-        description: ''
+        open: true
+        description: 'Summary fields determine how this person appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```
Comment (trackleft) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_person.default.yml:62]:
```suggestion
        description: 'Summary fields determine how this person appears in feeds, listings and other displays throughout the site.'
```

**Review by bberndt-uaz — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by joshuasosa — APPROVED** [review]

**Review by camikazegreen — APPROVED** [review]
I really like this change, I sometimes find myself forgetting to fill in the summary when creating News and discovering it only when seeing it show up in Views.

**Pull request merged**